README.md

```diff
@@ -56,20 +56,18 @@ npm i @aeternity/aepp-sdk@next
 npm i https://github.com/aeternity/aepp-sdk-js#develop
 ```
 
-### 2. Import (a chosen Flavor)
+### 2. Create a Keypair
```

Comment:
I think this could work, but only as point nr. 3 (of 4), with 4 being "Give Yourself some _AEs_" and 2 keep being "Import a Flavor", because choosing and Importing a flavor is still a basic entry point for those that start using the SDK.

What do you think @andrenarchy ?

Reply:
...or maybe we can switch poin nr. 2 with point nr. 3? (account first?)